Makefile

```diff
@@ -89,12 +89,8 @@ coverage: deps ## Generate code coverage report
 coverhtml: coverage ## Generate HTML coverage report
 	@go tool cover -html=coverage_cleaned.cov -o coverage_unit.html
 
-.PHONY: e2e-test-docker-image ## Build Docker image used in e2e tests
-e2e-test-docker-image: deploy/e2e-test-image/Dockerfile
-	@docker build -t $(E2E_TEST_DOCKER_IMAGE) deploy/e2e-test-image
-
```

Comment:
Should we keep these lines to keep the "knowledge" about building the docker image? In theory, I should be able to build the image locally and push it to the registry, right? I am unsure, we might also drop it (the line is not hard to remember and basic Docker knowledge) and local e2e tests will also fetch the remote image.